src/WebJobs.Extensions.Twilio/Config/TwilioSMSConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Twilio
+{
+    /// <summary>
+    /// Defines the configuration options for the Twilio binding.
+    /// </summary>
+    public class TwilioSmsConfiguration
```

Comment:
>TwilioSmsConfiguration [](start = 17, length = 22)

Some of our more recent changes have made the Configuration implement IExtensionConfigProvider -- it simplifies the classes a bit. (see DocDB for an example).